### ISSUE TITLE
Specify host error when cloning vm from template

### ISSUE THREAD

**Issue by MiloCheung** [fog/fog-vsphere#78]
I am trying to specify the host target on clone vm action, but I found that the cluster info is required if I specified the host target.

The vm_clone code(line: 135 to 139):

```
          if options.key?('host') && !options['host'].empty? && !cluster_name.nil?
            host = get_raw_host(options['host'], attributes[:cluster], options['datacenter'])
          else
            host = nil
          end
```

The question is how can I pass the cluster info into vm_clone method? It try to get the cluster info from "attributes" parameter. And I can't figure out that how to make the cluster into the "attributes" parameter. Should we change "attributes" to "options"? So that user can specify the host and cluster info from options. Thanks.

Error message: 
NameError: undefined local variable or method `attributes' for #<Fog::Compute::Vsphere::Real:0x007fef4946b988>
from /home/milo/.rvm/gems/ruby-2.1.2/gems/fog-vsphere-1.7.0/lib/fog/vsphere/requests/compute/vm_clone.rb:136
:in `vm_clone'

My sample code:
```
vm_options = {"name"=>"milo_test",
  "datacenter"=>"datacenter",
  "template_path"=>"Ubuntu Server 14.04",
  "server_image"=>"Ubuntu Server 14.04",
  "numCPUs"=>"1",
  "memoryMB"=>"1024",
  "power_on"=>false,
  "datastore"=>"datastore01",
  "resource_pool"=>["cluster", "resource_pool"],
  "host"=>"10.10.0.1"
}

clone_vm_result = client.vm_clone(clone_options)
```

**Comment by jrgarcia** [comment]
This was fixed by #76. I just released a new gem (1.7.1) with that code in it yesterday.

**Comment by jrgarcia** [comment]
Actually, try that and see if that fixes it, but I think this may be something else. Reopen this if that doesn't fix it and I'll take a look.
